src/python/WMCore/JobStateMachine/SummaryDB.py

```diff
@@ -83,12 +83,12 @@ def fwjr_parser(doc):
                             cmsRunCPUPerformance=pdict)
         if key.startswith('cmsRun'):
             perf = val['performance']
-            pdict['totalJobCPU'] += float(perf['cpu']['TotalJobCPU'])
-            pdict['totalJobTime'] += float(perf['cpu']['TotalJobTime'])
+            pdict['totalJobCPU'] += float(perf['cpu'].get('TotalJobCPU', 0))
```

Comment:
Alan, would it be worth to make it more safe by assigning dict as default value for `cpu`, e.g.
```
pdict['totalJobCPU'] += float(perf.get('cpu', {}).get('TotalJobCPU', 0))
```